src/mochada_kit/__init__.py

```diff
@@ -9,139 +9,10 @@
 @author: tgwoodcock
 """
 
```

Comment:
This `__init__` is massive! It would be good to divide this into three files: __init__.py, config.py, cli.py